### PR TITLE
Change grpc interface to compatible with brpc.

### PR DESCRIPTION
**Pull request by gongweibao**
A part of https://github.com/PaddlePaddle/Paddle/pull/11430

### CODE REVIEW

File: paddle/fluid/operators/distributed/grpc_sendrecvop_utils.cc [hunk 1]
```diff
@@ -0,0 +1,157 @@
+/* Copyright (c) 2016 PaddlePaddle Authors. All Rights Reserved.
```
Comment (typhoonzero) [inline paddle/fluid/operators/distributed/grpc_sendrecvop_utils.cc]: `utils` can not indicate what this file is doing, why not rename it to `grpc_serde.cc`?

Comment (gongweibao) [reply paddle/fluid/operators/distributed/grpc_sendrecvop_utils.cc]: Done.

File: paddle/fluid/operators/distributed/variable_response.h [hunk 2]
```diff
@@ -22,18 +22,35 @@
...
+  // ParseFrom needs to fall back to a more expensive parsing method.
+  // Every call must return a stream pointing at the beginning of
+  // the serialized RecvTensorResponse.
+  //
+  // Note that a subsequent call to contents() invalidates previous
+  // results of contents().
+  //
+  // Ownership of the returned stream is retained by the Source and
+  // should not be deleted by the caller.
+  virtual ::google::protobuf::io::ZeroCopyInputStream* contents() = 0;
```
Comment (typhoonzero) [inline paddle/fluid/operators/distributed/variable_response.h:51]: Seems this class is grpc specific?

**Review by typhoonzero — APPROVED** [review]
LGTM++ merge changes and expecting following refines.

File: paddle/fluid/operators/distributed/variable_response.h (hunk 2 above)
Comment (gongweibao) [reply paddle/fluid/operators/distributed/variable_response.h:51]: No, it's defined in protobuf.

**Pull request merged**